### PR TITLE
Bug 1345217 - Separate csv uploads to s3 from churn job

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This creates a separate notebook for uploading `s3://telemetry-parqet/churn/v2` as csv to s3. This separation of concerns makes it easier to reason about the data transformation process.

This notebook also gives more control over the locations of the csv files, and allows for easier removal of these files once depreciated.

### CODE REVIEW

**Comment by Dexterp37** [comment]
Anthony, is this going to replace the job I'm currently running?

**Comment by acmiyaguchi** [comment]
Yes, this will replace the job you are currently running.

File: etl/churn_to_csv.kp/knowledge.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        },
+        {
+            "name":   "Dashboard",
+            "bucket": "net-mozaws-prod-metrics-data",
+            "prefix": "telemetry-churn"
+        }
+    ]
+}
+
+if False:
```
Comment (Dexterp37) [inline etl/churn_to_csv.kp/knowledge.md]: Are you planning to leave this part in the notebook? If so, please leave a comment here stating that's just for debugging purposes and it's disabled.

**Comment by acmiyaguchi** [comment]
The CSV files are consistent as [this notebook shows](https://gist.github.com/acmiyaguchi/17bbead0f05c8233ad9a1d2fc94a44c5). 

I've added support for adding arbitrary dates to support backfilling too in this additional commit.

**Pull request merged**